### DISCUSSION TITLE
Error: ENOSPC: no space left on device, write

### DISCUSSION THREAD

**Discussion by wasurocks** [vercel/community#1370] [Help]
### Summary

I'm facing this error message which prevents the site from being deployed.
<img width="1191" alt="Screenshot 2566-01-27 at 18 19 28" src="https://user-images.githubusercontent.com/38748581/215074127-7999b2a4-add8-4046-a710-a3492887853c.png">

I read from https://vercel.com/docs/concepts/limits/overview#:~:text=Custom-,Disk%20Size%20(GB),-13 that a 13 GB size limit exists but when doing `du -d 1 -h` inside the site's repository (locally) after building, this is the result:
<img width="514" alt="Screenshot 2566-01-27 at 18 21 06" src="https://user-images.githubusercontent.com/38748581/215074517-ad7d2cca-b455-4b24-a90c-28d2f74df62c.png">

I also found similar issues, but it seems their sizes have really exceeded the limit (unlike what mine is showing):
- https://github.com/vercel/community/discussions/1293
- https://github.com/vercel/community/discussions/831

It does not exceed the limit, yet Vercel still shows that error.

I'd like to kindly ask to know how I could fix this, or if there is some solution that would take the minimum amount of steps.

Thank you!

**Comment by HeleneGrini** [comment]
I am facing the same issue. My build is only 1.7GB. I am able to deploy my site from my own machine using the cli, but this is really not optimal.  

What is the real issue here and how can it be fixed?  @vercel-infra 

Were you able to get it working @wasurocks ?

**Reply by wasurocks** [reply]
@HeleneGrini Nope, I ended up going with a static export hosted on firebase.
